src/Agent/NewRelic/Agent/Core/DataTransport/ConnectionManager.cs

```diff
@@ -107,12 +109,14 @@ private void Connect()
             {
                 HandleHttpErrorResponse(ex);
             }
+#if !NETFRAMEWORK // Only available in System.Net.Http
             // Occurs when the agent is unable to connect to APM. The request failed due to an underlying
             // issue such as network connectivity, DNS failure, server certificate validation or timeout.
             catch (HttpRequestException)
```

Comment:
It looks like we have a unit test that tests this path that needs to be updated.

Reply:
We do indeed!